metals-docs/src/main/scala/docs/RequirementsModifier.scala

```diff
@@ -20,15 +20,16 @@ class RequirementsModifier extends StringModifier {
     s"""
        |## Requirements
        |
-       |**OpenJDK or Oracle Java 8**. Eclipse OpenJ9 and Java 11 are not 
-       |supported, please make sure the `JAVA_HOME` environment variable 
+       |**OpenJDK or Oracle Java 8**. Eclipse OpenJ9 and Java 11 are not
+       |supported, please make sure the `JAVA_HOME` environment variable
        |points to valid Java 8 installation.
        |
        |**macOS, Linux or Windows**. Metals is developed on macOS and every PR is
        |tested on Ubuntu+Windows.
        |
-       |**Scala 2.12 and 2.11**. Metals works only with Scala versions $supportedScalaVersions.
-       |Note that 2.10.x and 2.13.x are not supported.
+       |**Scala 2.13 and 2.12**. Metals supports these Scala versions $supportedScalaVersions.
```

Comment:
Let's keep `Scala 2.13, 2.12 and 2.11` and add instead "Note that 2.11.x support is deprecated, it's recommended to upgrade to 2.12.".